.all-contributorsrc

```diff
@@ -0,0 +1,11 @@
+{
+  "projectName": "solidarity",
+  "projectOwner": "infinitered",
+  "files": [
+    "docs/existingContributors.md",
+    "README.md"
```

Comment:
This is only for updating the badge. But, the badge will need to be moved and updated to the HTML that all other badges are in.

That being said it looks like the `all-contributors-cli` supports updating the HTML formatted badge w/ the new count.